clients.asciidoc

```diff
@@ -454,7 +454,7 @@ While it is possible to fully sync the blockchain this way, the sync will take a
 
 Many Ethereum-based blockchains were the victim of a Denial-of-Service (DoS) attack at the end of 2016. Blockchains affected by this attack will tend to sync slowly when doing a full sync.
 
-For example, on Ethereum, a new client will make rapid progress until it reaches block 2,283,397. This block was mined on September 18th, 2016 and marks the beginning of the DoS attacks. From this block and until block 2,700,031 (November 26th, 2016), the validation of transactions becomes extremely slow, memory intensive, and I/O intensive. This results in validation times exceeding 1 minute per block. Ethereum implemented a series of upgrades, using hard forks, to address the underlying vulnerabilities that were exploited in the denial of service. These upgrades also cleaned up the blockchain by removing some 20 million empty accounts created by spam transactions.
+For example, on Ethereum, a new client will make rapid progress until it reaches block 2,283,397. This block was mined on September 18th, 2016 and marks the beginning of the DoS attacks. From this block and until block 2,700,031 (November 26th, 2016), the validation of transactions becomes extremely slow, memory intensive, and I/O intensive. This results in validation times exceeding 1 minute per block. Ethereum implemented a series of upgrades, using hard forks, to address the underlying vulnerabilities that were exploited in the denial of service. These upgrades also cleaned up the blockchain by removing some 20 million empty accounts created by spam transactions. <<[1]>>
```

Comment:
> These upgrades also cleaned up the blockchain by removing some 20 million empty accounts created by spam transactions.

This is technically slightly inaccurate (see my comment here: https://github.com/ethereumbook/ethereumbook/issues/477#issuecomment-383556405). I might reword this as, "by allowing the removal of...".